ihmc-high-level-behaviors/src/main/java/us/ihmc/behaviors/behaviorTree/BehaviorTreeNodeDefinition.java

```diff
@@ -121,8 +121,11 @@ public void setOnDiskFields()
 
    public void undoAllNontopologicalChanges()
    {
-      name.setValue(onDiskName);
-      notes.setValue(onDiskNotes);
+      if (isUndoAvailable())
+      {
+         name.setValue(onDiskName);
```

Comment:
The primary crash was name getting set to null, which native crashed an ImGui function.